core/src/main/scala/Config.scala

```diff
@@ -395,7 +471,12 @@ object Config {
     val nomadcfg = readNomad(cfg.subconfig("nelson.nomad"))
 
     val gitcfg = readGithub(cfg.subconfig("nelson.github"))
```

Comment:
Is configuration of github and gitlab under  `nelson.github`

Reply:
yep, for the time being. Which one is used is determined by the param `service`, e.g. 
```
service = "gitlab"
```
Default is `github` anyway.

I tried not to add many breaking changes in this first version of Gitlab support, but I think that we should re-think the algebra and the configuration to be more generic. Probably in another PR if you agree.